### ISSUE TITLE
[help] No 'transform' method in the softImpute class

### ISSUE THREAD

**Issue by charbelrseif** [iskandr/fancyimpute#121]
Hello, 

I am using the softImpute solver for a collaborative filtering problem.
After fitting and transforming my model on the training set using .fit_transform() I want to make predictions on the test set. However, the SoftImpute.transform method is not implemented!

Do you know how it could be implemented?

Thank you in advance.

**Comment by sergeyf** [comment]
Nope, sorry. Only `IterativeImputer` supports this unfortunately!
